release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -603,6 +603,21 @@ $ operator-sdk new <operator_name> --type=helm \
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874754[*BZ#1874754*])
 
+* The bare metal actuator is deleting the machine object when its underlying
```

Comment:
The OpenShift machine-api bare metal actuator currently deletes machine objects when the underlying host is deleted.  This behavior does not align with other cloud provider acutators and may result in the replacement machine not coming available due to a race condition with the bare metal API.

That's how I would phrase it, generally.  Make sure we link to bug and work around, other text is superfluous.

Reply:
@michaelgugino I made your suggested edits. The only thing I do not have is the workaround for this. Can you elaborate on what that is so I can include it? I didn't see anything in the BZ for that.

Reply:
I think the workaround is probably to follow [these steps](https://github.com/metal3-io/metal3-docs/blob/master/design/baremetal-operator/remove-host.md#steps) to remove a BareMetalHost.

FWIW I don't actually believe that there's a race between the Machine and bare metal APIs. AFAIK the only consequence is that the behaviour is inconsistent with other cloud providers, but maybe Michael can clarify what he meant there.